src/com/layout/col/col.js

```diff
@@ -0,0 +1,30 @@
+import {h, Component} from 'preact/preact';
+
+export default class Col extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render({children, flex = 1, flexGrow, flexShrink, flexBasis = 100}) {
```

Comment:
Prefer

```
render( props ) {
    let {flex = 1, flexGrow, flexShrink, flexBasis = 100} = props;
```

Reply:
This is the syntax I actually prefer. I was just following the syntax of the games page itself for state. Will change it to that.

Reply:
Yeah I only recently decided to change over (I think I've only discussed this with @local-minimum), so it's slowly being changed across the codebase.